tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -545,6 +545,9 @@ func (iptConfigurator *IptablesConfigurator) run() {
 	}
 
 	if iptConfigurator.cfg.InboundInterceptionMode == constants.TPROXY {
+		// rule to skip app ==> app traffic
+		iptConfigurator.iptables.InsertRuleV4(constants.PREROUTING, constants.MANGLE, 1, "-o", "lo", "-p", constants.TCP,
```

Comment:
It is not -o lo, it is -i lo. I forgot which i sent u.

Reply:
Ah ok. Let me fix that.

Reply:
Let me debug